tests/test_gp_var_dist.py

```diff
@@ -146,5 +146,176 @@ def cb(mod, i, loss):
 #     plt.savefig('figures/test_gp.png', bbox_inches='tight')
 #     plt.close()
 
+
+def test_GP_lat_prior():
```

Comment:
maybe we should just have a single function that does this with toeplitz on and off in a for loop?

Reply:
good point, I refactored so test_no_toeplitz_GP_lat_rior just calls the toeplitz one with a parameter to not use toeplitz.